source/index.md

```diff
@@ -975,6 +1015,13 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
+### Add credits resource
+
+Starting 2018-xx-xxTxx:xx:xx+00:00 the `/credits` resource has been added,
```

Comment:
IMO: `s/Starting 2018-xx-xxTxx:xx:xx+00:00 the/The/` and then we just don't deploy this PR before the implementation is in production.

We shouldn't forget also to communicate this explicitly to partners (that do credits).

Reply:
We have starting dates for a few other change announcements, so I will keep it. I acknowledge that a date is partly deducible from this PR (which is public), however, I think a changelog needs versions, dates or both in order to be usable on its own.